### PR TITLE
[mac-install.md] Switched from gibMacOS to Munki

### PR DESCRIPTION
**Pull request by dreamwhite**
Hi all ^^
Despite gibMacOS has served us well for many months, probably the wisest way to download macOS from a macOS environment is using Munki's script.
gibMacOS is based off of Munki's script but I feel it as less comfortable:

- with gibMacOS you have to build the installer
- with Munki's script, the installer is automatically built and bundled inside a DMG (way more user friendly than gibMacOS)

For the end user, handling the command line is way more difficult (despite it being just a folder copy-n-paste ._.") than just executing a single command and waiting for it to finish.

Please let me know if you're interested in this PR and what are the necessary adjustment I need to apply.

Special thanks go to:

- @1alessandro1 who suggested this script a few months ago
- @Emanuele-1998 for testing with multiple versions of macOS (mainly High Sierra, Mojave and Catalina)
- @marianopela for helping a poor neapolitan guy writing in english xD 

Best regards
dreamwhite

### CODE REVIEW

File: installer-guide/mac-install.md [hunk 1]
```diff
@@ -5,73 +5,40 @@
...
 From a macOS machine that meets the requirements of the OS version you want to install, go directly to the App Store and download the desired OS release and continue to [**Setting up the installer**](#setting-up-the-installer).
 
-For machines that need a specific OS release or can't download from the App Store, you can use the gibMacOS utility.
+For machines that need a specific OS release or can't download from the App Store, you can use the Munki's InstallInstallMacOS utility.
 
-Now let's grab [gibMacOS](https://github.com/corpnewt/gibMacOS), then unzip it into a local directory.
+In order to run it, just copy and paste the below command in a terminal window:
 
-Next run the `gibMacOS.command`:
+`curl -O https://raw.githubusercontent.com/munki/macadmin-scripts/main/installinstallmacos.py && sudo /usr/bin/python installinstallmacos.py`
```
Comment (khronokernel) [inline installer-guide/mac-install.md:20]: Few things:

* Better to use the global `python` command for up-to-date binary, as  `/usr/bin/python` won't always have the latest
  * more a nitpick as I see the script is both py2 and py3 compliant anyways
* Is `sudo` needed for this? Not a fan of giving elevated privileges unless required
* Another recommendation would be to make a new folder and run there, so no need to dirty the user folder(ie. `mkdir macOS-Installers`)
  * Mostly nitpick again, not that important
* Incorrect usage of code block, please use triple backquote's instead: ```

File: installer-guide/mac-install.md [hunk 2]
```diff
@@ -5,73 +5,40 @@
...
-  * For 10.15 and older
-
-### Extracting the installer
-
-For macOS 11 and newer, Apple now packages up the installer into the InstallAssistant package. This will be located under `gibMacOS/macOS Downloads/`:
-
-![](../images/extras/big-sur/readme/final-download.png)
-
-Run the InstallAssistant.pkg and point this to whichever drive you're booting off of, this is where the Install.app will be dropped:
+Once finished, you'll find in your `Users` folder a DMG containing the macOS Installer, called like `Install_macOS_11.1-20C69.dmg`. Mount it and you'll find the installer application.
```
Comment (khronokernel) [inline installer-guide/mac-install.md:35]: As mentioned earlier, maybe don't use the base `~/` folder to keep things cleaner

File: installer-guide/mac-install.md [hunk 3]
```diff
@@ -316,6 +283,10 @@ You can also replace the `createinstallmedia` path with that of where your insta
 Pulled from Apple's own site: [How to create a bootable installer for macOS](https://support.apple.com/en-us/HT201372)
 
 ```sh
+
```
Comment (khronokernel) [inline installer-guide/mac-install.md:286]: Please remove extra tab

File: installer-guide/mac-install.md [hunk 4]
```diff
@@ -5,73 +5,40 @@
...
-
-![](../images/installer-guide/mac-install-md/gib-location.png)
-
-You will be prompted for the macOS installer files which were downloaded to `macOS Downloads` folder in the gibMacOS directory.
-
-From the Finder, drill down to the folder containing the downloaded files and either drag it to the command line or "Cmd+C" and paste it to the terminal.
-
-Once the task is completed, exit the utility. You will find the Install file in the directory.
-
-Move the newly created image to the Applications folder – this will simplify the next section.
```
Comment (khronokernel) [inline installer-guide/mac-install.md:69]: I recommend still telling users to move it to the `/Applications` folder as users will try to execute `createinstallmedia` without correcting the path

Other solution would be to edit the below `createinstallmedia` commands, however would get complicated for people with installers pulled directly from the App Store or Apple's archival site

**Comment by khronokernel** [comment]
From a quick peak at the script, looks *mostly* fine to me. Not a fan of the DMG but that's subjective.

My main concerns with this PR:

* Images are quite low res(subjective, I can always remake them later)
* You deleted old images(We use images across the guide, not just a single page. Don't delete unless you are absolutely sure these images are unused elsewhere)
* Linting failed
* Other concerns are commented in the PR

If you can fix those issues, I'm more than happy to merge. Thanks for the PR!

**Comment by khronokernel** [comment]
Oh and for spellcheck, just let it fail. Our shared build-files idea is likely going to be reverted, so that'll be something I'll fix

File: installer-guide/mac-install.md (hunk 1 above)
Comment (khronokernel) [reply installer-guide/mac-install.md:20]: Tested, yeah `sudo`'s needed

File: installer-guide/mac-install.md (hunk 2 above)
Comment (1alessandro1) [reply installer-guide/mac-install.md:35]: The developer recommends to use `/Users/Shared` or similar

Comment (khronokernel) [reply installer-guide/mac-install.md:35]: User's base folder(`~/`) isn't protected, so macOS shouldn't care either

**Comment by khronokernel** [comment]
Had some free time, so resolved the issues here:

* https://github.com/dortania/OpenCore-Install-Guide/pull/222

Thanks so much for the PR @1alessandro1, really appreciate the help!

**Comment by dreamwhite** [comment]
@khronokernel Sorry for not replying but I was busy for a while (a big while 😂). I really appreciate that you approved my pull request. I hope you enjoyed it. 
Please let me know whether you need any type of fix inside the method.
Kindly,
dreamwhite

**Pull request closed without merging**